src/views/dashboards/Home/index.vue

```diff
@@ -75,6 +75,13 @@ export default {
     handleCrumbClick(crumb) {
       this.$router.push(crumb.path);
     },
+    goToCardDetails(cardId, cardTitle) {
+      const path = `/dashboards/${cardId}`;
+      this.$router.push({
+        path: path,
+        query: { title: cardTitle },
+      });
+    },
```

Comment:
```suggestion
      this.$router.push({
        name: 'dashboard',
        params: { id: cardId } }
        query: { title: cardTitle },
      });
    },
```

This way of doing a push is more maintainable because the route name tends to change less than the path.